recipes/go-recipes/priority.go

```diff
@@ -66,20 +66,20 @@ func (prty Priority) Push(trtr fdb.Transactor, value interface{}, priority int)
 
 func (prty Priority) _NextCount(trtr fdb.Transactor, priority int) int {
 	res, err := trtr.Transact(func(tr fdb.Transaction) (interface{}, error) {
-		kr, e := fdb.PrefixRange(prty.PrioritySS.Pack(tuple.Tuple{priority}))
-		if e != nil {
-			return nil, e
+		kr, err := fdb.PrefixRange(prty.PrioritySS.Pack(tuple.Tuple{priority}))
+		if err != nil {
+			return nil, err
 		}
 
-		ks, e := tr.Snapshot().GetRange(kr, fdb.RangeOptions{1, -1, true}).GetSliceWithError()
-		if e != nil {
-			return nil, e
+		ks, err := tr.Snapshot().GetRange(kr, fdb.RangeOptions{1, -1, true}).GetSliceWithError()
+		if err != nil {
+			return nil, err
 		}
 
 		if len(ks) == 0 {
 			return 0, nil
 		}
-		k, e := prty.PrioritySS.Unpack(ks[0].Key)
+		k, err := prty.PrioritySS.Unpack(ks[0].Key)
```

Comment:
`err` here is ignored, should be evaluated.

Reply:
i couldn't get it. could you come up once again.

Reply:
Sure: I think this `err` is never returned to caller, so effectively is ignored.
But it's not something introduced by your PR, it just becomes evident due to the changes.

Reply:
Assuming that `k` will be nil in the event of an error, that could cause an nil pointer exception.